src/service.js

```diff
@@ -206,11 +206,13 @@
 
                 /**
                  * Executes the reCaptcha
+                 * @param widgetId
+                 * @param [action] the action for reCAPTCHA v3
                  */
-                execute: function (widgetId) {
+                execute: function (widgetId, action) {
```

Comment:
recaptcha expects that the provided action is a string. Do we know that it tollerates `undefined` ?